source/quick-start-rails/write-data.txt

```diff
@@ -0,0 +1,41 @@
+.. _mongoid-quick-start-rails-write-data:
+
+=====================
+Write Data to MongoDB
+=====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: test connection, runnable, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Create a new restaurant
+
+      In your browser at http://127.0.2.2:3000/restaurants, you can
+      scroll to the bottom of the list and click the :guilabel:`New
+      restaurant` link to navigate to the ``/restaurants/new`` route. On
```

Comment:
Q: confirming 'restaurant' is lowercase here but capitalized in 'Create Restaurant' later

Reply:
Yes, it's part of the base template shrug